.vscode/settings.json

```diff
@@ -0,0 +1,21 @@
+{
+  "eslint.format.enable": true,
+  // "eslint.trace.server": "verbose",
```

Comment:
Probably delete this lol

Reply:
I kept it in case there are issues in the future, I won't have to look this up again

Reply:
Oh you mean delete the comment and actually have it be verbose all the time?

Reply:
No, just entire remove, since technically `//` isn't even valid JSON (though VSCode appears to tolerate it)